integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DeployUtil.java

```diff
@@ -227,21 +227,24 @@ private static void createDeployJob(String deployScriptConfigMap, String namespa
 
   }
 
-
   /**
-   * Deploy application to a cluster using REST API with curl utility.
+   * Deploy application to a set of target using REST API with curl utility.
+   * Note the targets parameter should be a string with following format
+   *  { identity: [ clusters, 'cluster-1' ] }   for cluster target 
```

Comment:
can you document how to deploy to multiple clusters and servers?

Reply:
Updated the javadoc as follows  ...

Deploy application to a set of target using REST API with curl utility.
Note the targets parameter should be a string with following format
  {identity: [clusters,'cluster-1']}   for cluster target 
  {identity: [servers, 'admin-server']} for server target 
  OR a combination for multiple targets, for example 
   {identity: [clusters,'mycluster']}, {identity: [servers,'admin-server']}